bait-files/src/main/webapp/userHistory.js

```diff
@@ -0,0 +1,52 @@
+const url =
+    'https://step-bait-project-2020.uc.r.appspot.com/review.html?reviewId=';
+
+function getCompletedReviews() {
+  fetch('/userHistory?is-pending-review=false')
+      .then(response => response.json())
+      .then((reviews) => {
+        for (review of reviews) {
+          addReview(review, document.getElementById('completed-table'));
+        }
+      });
+}
+
+function getPendingReviews() {
+  fetch('/userHistory?is-pending-review=true')
+      .then(response => response.json())
+      .then((reviews) => {
+        for (review of reviews) {
+          addReview(review, document.getElementById('pending-table'));
+        }
+      });
+}
+
+function addReview(review, tableToAddTo) {
+  const queue = tableToAddTo;
```

Comment:
Why create this local variable at all?  Looks like you're just doing one thing with it at the very end of the function, and you could just call tableToAddTo.append(row); instead of creating "queue".

Reply:
Removed